lib/src/base-styles/_nx-responsive.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+@media (max-width: 1365px) {
+  .nx-body {
+    overflow: auto;
```

Comment:
Does this really need a media query?  Wouldn't it work to just always have this style in conjunction with the `min-width` and `max-width`?

Reply:
> Does this really need a media query? Wouldn't it work to just always have this style in conjunction with the `min-width` and `max-width`?

Not only do we not need the media query, we don't even need the `overflow: hidden` value that it overrides. https://github.com/sonatype/sonatype-react-shared-components/pull/185/commits/8d95298f9b3e84036d29fe26a1dbdc521051aebc